std/compress/io.go

```diff
@@ -0,0 +1,103 @@
+package compress
+
+import (
+	"bytes"
+	"compress/gzip"
+	"github.com/consensys/gnark/frontend"
+	"golang.org/x/exp/constraints"
+	"io"
+	"os"
+)
+
+func GzWrite(outFileName string, o io.WriterTo) error {
```

Comment:
why do we need to expose that exactly? if you only need it in test, put it iin _test.go files 👍 (but I don't understand from the code why we need it)

Reply:
Currently using it for code that outputs the compiled cs, which has to run as a standalone application and not a test.